sdk/src/test/scala/com/horizen/fixtures/SidechainNodeViewHolderFixture.scala

```diff
@@ -96,11 +95,13 @@ trait SidechainNodeViewHolderFixture
   val sidechainStateStorage = new SidechainStateStorage(getStorage(), sidechainBoxesCompanion)
   val sidechainStateForgerBoxStorage = new SidechainStateForgerBoxStorage(getStorage())
   val sidechainStateUtxoMerkleTreeStorage = new SidechainStateUtxoMerkleTreeStorage(getStorage())
+  val sidechainStateUtxoMerkleTreeProvider: SidechainUtxoMerkleTreeProviderImpl = SidechainUtxoMerkleTreeProviderImpl(sidechainStateUtxoMerkleTreeStorage)
```

Comment:
This is a good point. In theory, the tests using these objects should not be impacted by which of the 2 classes is used, so I could have used also the class without CSW (or even a mock object). In practice, I preferred to use the class with CSW, in order to be sure that existing tests still worked as before, because you never know. In addition, in my understanding, CSW enabled should be the default so, everything being equal, I prefer having the tests with the CSW enabled. 
To be 100% sure, I should have duplicated all the tests in order to use both classes, this would be the only way to be sure everything is actually fine, but I thought it was too much and I made a compromise.